config/locales/ja.yml

```diff
@@ -554,6 +554,7 @@ ja:
       balance_pending: 保留中の支払い
       next_deposit_date: "次の入金日："
       payout_in_progress: "進行中の支払い"
+      payout_minimum_balance: "Minimum balance for payout is %{amount} %{currency}"
```

Comment:
Reminder to get the japanese locale before merge. :)

Reply:
I'm aware :)

Reply:
Do you have the locales?

Reply:
```suggestion
      payout_minimum_balance: "最低出金金額は %{amount} %{currency}です"
```